Source/WebCore/accessibility/AXTextMarker.cpp

```diff
@@ -607,17 +607,18 @@ String AXTextMarkerRange::toString() const
 
         StringBuilder builder;
         for (; !it.atEnd(); it.advance()) {
+            RefPtr countedNode = it.node();
```

Comment:
Nit: WebKit style convention is to simply use the plain-old variable name. No `counted` or anything. In special instances (like when capturing `this` in a lambda), we may use `weakThis` or `protectedThis`. But not for normal variables. You won't see any examples of `RefPtr counted*` in the codebase. We prefer:

`RefPtr node = it.node();`